lua/rtt.cpp

```diff
@@ -1944,6 +1974,7 @@ static const struct luaL_Reg Service_f [] = {
 	{ "getOperationNames", Service_getOperationNames },
 	{ "hasOperation", Service_hasOperation },
 	{ "getPortNames", Service_getPortNames },
+	{ "getOperationInfo", Service_getOperationInfo },
```

Comment:
Although I like the name `getOperationInfo` more I would propose to keep the `Service` and `TaskContext` Lua interfaces the same, where applicable. Could you either rename this metatable entry to `"getOpInfo"` or add an alias `"getOperationInfo"` for TaskContext?

The order the methods are listed is different in `Service_f` and `Service_m`. I am not sure if this is important for `luaL_register` or purely cosmetic.